### DISCUSSION TITLE
How to remove periodic tasks for Celery beat schedule?

### DISCUSSION THREAD

**Discussion by mrn-aglic** [celery/celery#8601] [Q&A]
Hi, in our project we're using Celery with Redis. We depend a lot on periodic tasks. 
I'm interested in how we could completely remove periodic tasks from being executed? 

I read that there is this: https://docs.celeryq.dev/en/stable/userguide/workers.html#revoke-revoking-tasks

However, I want to be able to revoke tasks by name if possible. My priority is not to cancel the currently running task, but to prevent those periodic tasks from running again. 

It may also be the case that once a task is removed, we want to add it as a periodic task later on.

**Comment by Alekky09** [comment]
Why would you stop a periodic task from being executed? These are not meant to be stopped like that, that's why they're called "periodic", because they operate on a schedule.
You may want to implement this differently, or change their cron schedule. If you're using a Database Scheduler, you can turn off the task in the DB manually too.

**Reply by mrn-aglic** [reply]
Because there is a need for running the tasks every day for different configurations and I want to allow the user to add and remove these configurations themselves. 
Of course there is the option of reading the list of configurations and scheduling them with eta... but disabling/removing periodic tasks seems like a cleaner solution. 
But django celery beat could be the way to go.
